packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -391,8 +391,6 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         //   2. The amount of gas provided to the call to the target contract is at least the gas
         //      limit specified by the user. If there is not enough gas in the callframe to
         //      accomplish this, `callWithMinGas` will revert.
-        // Additionally, if there is not enough gas remaining to complete the execution after the
-        // call returns, this function will revert.
```

Comment:
It's slightly unclear to me if this is meant to indicate that this is handled by `SafeCall.callWithMinGas` or if it's just friendly reminder about how the EVM works. 

Also I don't think it'd be obvious to an outsider what the purpose of this statement is. 

Maybe it would help to append it with: 

```
This ensure that calls which do not provide sufficient additional gas will not finalize
the withdrawal, so that it can be attempted again.
```

Honestly maybe I'm just being pedantic and there's a lot of words here already, so I'm not going to block over this.

Reply:
This comment was removed because the `callWithMinGas` function will **not** revert if there is not enough gas remaining to complete the rest of `finalizeWithdrawalTransaction` after the call is performed. In the case that there's not enough gas to complete execution, `finalizeWithdrawalTransaction` will OOG, allowing for the caller to retry with a higher gaslimit.

Reply:
This comment was also wrong prior to this change - The behavior of `callWithMinGas` has never attempted to reserve any more gas than was required to forward `_minGasLimit` to the target context.

Reply:
OMG, I was reviewing that comment as if it was an addition. 🫠